### PR TITLE
mubert-base.joda-time.mid-1142.idx-240716.10.mutant

### PR DESCRIPTION
**Pull request by Mariam22-hub**

### CODE REVIEW

**Comment by faustinoaq** [comment]
This changes code and also adds new empty line

Needs more review

File: joda-time/src/main/java/org/joda/time/chrono/ISOYearOfEraDateTimeField.java [hunk 1]
```diff
@@ -56,8 +56,9 @@ public DurationField getRangeDurationField() {
     @Override
     public int get(long instant) {
         int year = getWrappedField().get(instant);
-        return year < 0 ? -year : year;
+        return year <  1 ? -year : year;
```
Comment (faustinoaq) [inline joda-time/src/main/java/org/joda/time/chrono/ISOYearOfEraDateTimeField.java:59]: Is this change actually required?

**Review by faustinoaq — CHANGES_REQUESTED** [review]

File: joda-time/src/main/java/org/joda/time/chrono/ISOYearOfEraDateTimeField.java (hunk 1 above)
Comment (darko-marinov) [reply joda-time/src/main/java/org/joda/time/chrono/ISOYearOfEraDateTimeField.java:59]: No, please change just the formatting not the logic.

@Mariam22-hub please address this in the next 2-3 days, or we'll assign to someone else.
